language_detector/main.py

```diff
@@ -1,9 +1,53 @@
 """This is the entry point of the program."""
 
-from .languages import LANGUAGES
+try:
+    from .languages import LANGUAGES
+except:
+    from languages import LANGUAGES
 
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    spanish = []
+    english = []
+    german = []
```

Comment:
You're hard coding the list of languages, and initializing them all here individually.  I think you would be better off wrapping them up in a single data structure like a dict.  This also makes comparison of the totals later easier.